debian/control

```diff
@@ -29,3 +29,14 @@ Description: high performance general RPC framework
  transparently, and makes it easier to build connected systems.
  .
  This package provides the gRPC C bindings.
+
+Package: libgrpc0-cil
+Architecture: any
+Section: cli-mono
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
I think the libgrpc_csharp_ext.so depends on libgrpc and libgpr but dependencies don't state that.